### PR TITLE
NodeJS - Intro to Express: Add video link to additional resources

### PR DESCRIPTION
**Pull request by gregolive**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Web Dev Simplified's video ***Learn Express JS In 35 Minutes*** was my most helpful resource when learning Express due to the large amount information provided in a relatively short (35 minute) video. I think this video is worth adding to the <i>Additional Resources</i> section of the **Introduction to Express** NodeJS lesson because it could be a useful resource for others as well.

**2. This PR:**

Adds a new video resource to the <i>Additional Resources</i> section of NodeJS - **Introduction to Express**.

**3. Additional Information:**

### CODE REVIEW

File: nodeJS/express_and_mongoose/introduction_to_express.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ This section contains helpful links to other content. It isn't required, so cons
 
 - The book [Express in Action](https://www.manning.com/books/express-in-action?a_bid=fe3fcff7&a_aid=express-in-action) by Evan M. Hahn is an incredible resource for Express beginners.
 - [This](https://www.youtube.com/watch?v=L72fhGm1tfE) video crash course provides you with all the basic concepts.
+- Web Dev Simplified's [Express JS crash course](https://www.youtube.com/results?search_query=web+dev+simplified+node) also packs a ton of great information into a 35 minute video.
```
Comment (01zulfi) [inline nodeJS/express_and_mongoose/introduction_to_express.md:42]: This link opens up a Youtube search result. Can you link the video you're referring to?

Comment (gregolive) [reply nodeJS/express_and_mongoose/introduction_to_express.md]: @01zulfi Apologies for that. I have updated the link now I believe. Thanks for your help!

**Review by 01zulfi — APPROVED** [review]
Looks good :rocket:

**Comment by gregolive** [comment]
> Looks good 🚀

Thank you for the help!

**Pull request merged**